docs/learning/tutorials/develop-desktop-viewer.md

```diff
@@ -1,36 +1,46 @@
 
 ## Setup
 
-- [Install necessary prerequisites]($docs/getting-started/development-prerequisites)
-- [Clone desktop-starter repo](https://github.com/imodeljs/desktop-starter)
+- [Install necessary prerequisites]($docs/learning/tutorials/development-prerequisites).
+- From a terminal, `npx create-react-app your-app-name --template @itwin/desktop-viewer --scripts-version @bentley/react-scripts`
+  - This will generate a new application based on the iTwin Viewer React component in the `your-app-name` directory.
+- Open the `your-app-name` directory in VS Code
+- Follow these steps to obtain a new OIDC client to use in your viewer application:
+  1. Go to <https://developer.bentley.com>
+  2. Click the **Sign In** button and sign-in using your Bentley account credentials
+      - If you have not already registered, click **Register now** and complete the registration process.
+  3. Navigate to the [My Apps](https://developer.bentley.com/my-apps/) page
+  4. Click the **Register New** button
+  5. Give your application a Name
+  6. Select the **Visualization** API
+  7. Select application type **Desktop**
+  8. Enter **Redirect URL** `http://localhost:3000/signin-callback`
+  9. Enter **Post logout Redirect URL**: `http://localhost:3000`.
+  10. Click the **Save** button
+
+Once your new application is saved a client ID is generated, add the client ID and redirect uri to the the following variables in the `.env` file within the application's root directory: `ITWIN_VIEWER_CLIENT_ID` and `ITWIN_VIEWER_REDIRECT_URI`.
+
+- From a terminal at your application's root directory, `npm start`. This will serve the application with live reloading.
+
+> Note: The live reloading is for the frontend only. The backend would need to be re-built on each change using `npm run build:backend`.
 
-## Create application
+## Build
 
-- Navigate to [developer.bentley.com](https://developer.bentley.com/register/)
-- Sign in
-- Configure application
-  - Provide application name
-  - API associations: "Visualization"
-  - Application type: "Desktop/Mobile"
-  - Redirect URI: `http://localhost:3000/signin-callback`
-  - Post logout redirect URI: `http://localhost:3000`
-  - Save
-- Click on the name of the client you just created to see the details
-- Copy Client ID to your clipboard
+From a terminal at your application's root directory, `npm run build`. This will create both the frontend in the "build" folder and backend in the "lib" folder within the application's root directory. It is not necessary to build the application during development.
 
-## Build
+The app will open a pre-packaged offline snapshot iModel. You will likely want to use your own snapshot iModel. Follow the [Create a snapshot iModel]($docs/learning/tutorials/create-test-imodel-offline) tutorial. Then open your snapshot by clicking the Home button and selecting "Open Offline Snapshot". Alternatively, set the absolute path of your snapshot in `.env.local` prior to running `npm run start`.

```

Comment:
> "set the absolute path of your snapshot in .env.local"

What is the name of the var?

Reply:
`ITWIN_VIEWER_SNAPSHOT`

Reply:
to be fair a lot of these steps are in the readme of the applicvation that gets bootstrapped.

Reply:
Added

Reply:
Ok. I would assume a user would likely switch focus to the code/README once they've bootstrapped anyway. Fine as is, I suppose.

Reply:
Yes, I pushed this change but I do think our first indirection (from the getting started page) and then a duplicate copy of this page's content in the READMEs is not the best experience...  Lets see what we can do about that in another PR.